CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -227,93 +229,52 @@ function Uninstall-Agent {
     }
 }
 
-function Assert-IsAgentServiceEnabled {
+function Get-AgentServiceStatus {
     Param ([Parameter(Mandatory = $true)] [PSSessionT] $Session)
-
-    $MaxWaitTimeInSeconds = $MAX_WAIT_TIME_FOR_AGENT_IN_SECONDS
-    $TimeBetweenChecksInSeconds = $TIME_BETWEEN_AGENT_CHECKS_IN_SECONDS
-    $MaxNumberOfChecks = [Math]::Ceiling($MaxWaitTimeInSeconds / $TimeBetweenChecksInSeconds)
-
-    for ($RetryNum = $MaxNumberOfChecks; $RetryNum -gt 0; $RetryNum--) {
-        $Status = Invoke-Command -Session $Session -ScriptBlock {
-            return $((Get-Service "ContrailAgent" -ErrorAction SilentlyContinue).Status)
-        }
-        if ($Status.Value -eq "Running") {
-            return
-        }
-
-        Start-Sleep -s $TimeBetweenChecksInSeconds
+    $Status = Invoke-Command -Session $Session -ScriptBlock {
+        return $((Get-Service "ContrailAgent").Status)
```

Comment:
Unnecessary `$()`. I think this `$()` should only be used in string interpolation. Outside a string, it's a no-op.

(The `return` and temporary `$Status` are also unnecessary). Remember, PS is quite an expression oriented language.